### PR TITLE
Fix toggling of directory previews

### PR DESCRIPTION
**Pull request by maidl0ver**
A very simple quick fix to make it possible to disable directory previews after enabling them.

### CODE REVIEW

File: eval.go [hunk 1]
```diff
@@ -48,6 +48,10 @@ func (e *setExpr) eval(app *app, args []string) {
 		app.ui.loadFile(app, true)
 	case "dirpreviews":
 		gOpts.dirpreviews = true
+	case "nodirpreviews":
+		gOpts.dirpreviews = false
+	case "dirpreviews!":
+		gOpts.dirpreviews = false
```
Comment (ilyagr) [inline eval.go:54]: This should say `gOpts.dirpreviews = !gOpts.dirpreviews`.

Comment (maidl0ver) [reply eval.go]: Indeed, thank you for noticing. I wonder how I let that slip through...

File: complete.go [hunk 2]
```diff
@@ -124,6 +124,8 @@ var (
 		"dirfirst!",
 		"dironly",
 		"dirpreviews",
+		"nodirpreviews",
+		"dirpreviews!",
 		"nodironly",
 		"dironly!",
```
Comment (p-ouellette) [inline complete.go:130]: nit: move nodironly and dironly! up three lines, after dironly

Comment (maidl0ver) [reply complete.go]: That's fair, they are sorted properly now

**Comment by gokcehan** [comment]
@maidl0ver Looks good thanks for the patch.

**Pull request merged**